docs/validate-stable-rc-builds.md

```diff
@@ -0,0 +1,178 @@
+Greg KH pushes on to stable rc trees and creates a tag
+then sends out email on stable ML with the list of patches included in that
+branch.
+
+You may check your inbox with give filters to identify the stable rc review
+for a given branch.
+
+## Build Test
+### Check for build is PASS
+open lkft.linaro.org website and check the build status
+From lkft.linaro.org you will re-directed to each branch builds for
+each device. If you see build status is in progress wait for the
+build to finish.
+
+https://ci.linaro.org/job/openembedded-lkft-linux-stable-rc-4.9/
+
+### Report build failure
+If build FAIL report the build failure along with build error log on
+respective email thread. build error should be in plain text and include
+branch, head commit id and architecture device.
+
+### Bisecting bad commit
+git bisect is helpful for finding the bad commit
+If you could track down and find bad commit causing build failure then
+report the detailed email on stable rc ML and include patch author and
+Sub-system maintainers.
+
+### Cancel multiple pushes
+If you see multiple pushes in short period of time on a given branch.
+Cancel the old build jobs on ci.linaro.org
+
+## LAB ready
+Check all devices status on lkft.linaro.org instance,
+make sure you have good number of devices are online to finish the testing
+in time. If devices are offline check with LAB team and know the reason for
+offline or maintenance mode
+
+Ping LAB team on IRC and report device related problem.
+sendout email to LAB team
+Create a critical Ticket if needed.
+
+Example LAB problems,
+Mounting external Harddrive might be failures for Juno-r2 / x86_64.
+Mouting NFS rootfs server might be a problem.
+Serial connection problem.
+LAVA dispatcher problem.
+
+### Sanity test
+For each branch build testing the sanity test will be done and these
+sanity jobs have high priority and runs LTP ltp-quickhit-tests jobs on
+4 architecture's devices arm64, arm, x86_64 and i386.
+
+Check dmesg and ensure no kernel panic, BUG, warning in boot log and test log.
+If you find Kernel panic or warnings please report on stable ML
+If you find boot failure with or without kernel crash log please report on
+stable ML.
+Check each test results and ensure no failures
+If any find any test failure, try to reproduce the test failure with possible
+minimal steps and report failure on stable ML include respective mailing lists
+for example LTP / kselfest / netdev / bpf mailing lists 
+
+https://qa-reports.linaro.org/lkft/linux-stable-rc-4.9-oe-sanity
+
+### Validate each branch results
+Each stable rc branch have defined LAVA test job priority.
+Please refer doc/lava-job-priority.md file for more details.
+
+The 4.9 branch have high priority so check 4.9 branch jobs are getting
+finished first and look for the test finished count on
+
+https://qa-reports.linaro.org/lkft/linux-stable-rc-4.9-oe/
+
+The each device submits 16 common jobs and few special test suites
+
+#### Number of tests
```

Comment:
For this section and the next two, I'm afraid they'll go out of date fast and there may be a better place to refer to the data. (I'm not certain where, though we do have some on lkft.linaro.org.

Reply:
For easy reference I would like to keep list of devices, test names and number of tests.